package.json

```diff
@@ -19,18 +19,18 @@
     "package": "npm run build && node scripts/create-packages.js"
   },
   "devDependencies": {
-    "@vuepress/plugin-google-analytics": "^1.8.2",
-    "archiver": "^5.3.0",
-    "chart.js": "^2.9.4",
-    "eslint": "^7.23.0",
-    "eslint-config-chartjs": "^0.3.0",
-    "rollup": "^2.44.0",
-    "rollup-plugin-terser": "^7.0.2",
-    "vuepress": "^1.8.2",
-    "vuepress-plugin-redirect": "^1.2.5",
-    "vuepress-theme-chartjs": "^0.2.0"
+    "@vuepress/plugin-google-analytics": "1.8.2",
```

Comment:
I would not use strict versions but keep it the way it was before. The method described in [my comment](https://github.com/chartjs/chartjs-plugin-deferred/pull/21#issuecomment-996499912) correctly generates the lock file for me. You may also need to delete the `node_modules` folder, else it will still use the already installed versions:

```sh
- npm -v                         # should be ~6.14.x
- rm package-lock.json
- npx rimraf node_modules
- npm install
```

`package-lock.json` should contain just a few changes related to chart.js v3 (even less than your last commit).

Reply:
I can't get it to work the way you like it too, even changed to my linux install to see if that would fix it but it also changes the checksums to sha512 from sha1.

Used npm: 6.14.15 and node: 14.17.2 with the commands you described above.

Reply:
Indeed, you actually need to revert to the `package-lock.json` that is in master before running `npm install`. I will fix it and push to your branch.